demo/README.md

```diff
@@ -82,41 +83,64 @@ http_proxy_port: 900
+# Running Demo using Reseller
 
-# Running Demo
+The FIDO IoT Reseller Sample can be used in different ways depending on its positioning in the Supply-Chain.
+In the following use-case, the FIDO IoT Manufacturer provisions the Device using DI and sells the Device to FIDO IoT Reseller, that later sells the Device to the FIDO IoT Owner, that completes the device onboarding (TO0 and TO2):
 
 1. Start the FIDO IoT Manufacturer Sample as per the steps outlined in [Manufacturer README](manufacturer/README.md).
 
 2. Complete Device Initialization (DI) by starting the FIDO IoT HTTP Java Device Sample as per the steps outlined in [Device README](device/README.md). Delete any existing 'credential.bin' before starting the device.
 
-3. Complete Ownership Voucher Extension by using the API `GET /api/v1/vouchers/<serial_no>` and save the Ownership Voucher. By default, existing customer with customer Id '1', is assigned to the device. To add a new customer and assign the inserted customer to the device, please refer to [FIDO IoT Manufacturer REST APIs](manufacturer/README.md/#fido-iot-manufacturer-rest-apis) for more information about the API.
+3. Complete Ownership Voucher Extension by using the API `GET /api/v1/vouchers/<serial_no>` and save the Ownership Voucher. Assign customer with customer Id '2' to the device. Please refer to [FIDO IoT Manufacturer REST APIs](manufacturer/README.md/#fido-iot-manufacturer-rest-apis) for more information about assigning customer to a device.
 
-4. Start the FIDO IoT RV Sample as per the steps outlined in [RV README](rv/README.md).
+4. Start the FIDO IoT Reseller Sample as per the steps outlined in [Reseller README](reseller/README.md).
 
-5. Start the FIDO IoT Owner Sample as per the steps outlined in [Owner README](owner/README.md). Import the extended Ownership voucher from Step#3 into the Owner database by using the API `POST /api/v1/owner/vouchers/`. Please refer to [FIDO IoT Owner REST APIs](owner/README.md/#fido-iot-owner-rest-apis) for more information about the API. Optionally, if service info transfer is needed, please refer to [Enabling Service info transfer](#enabling-service-info-transfer).
+5. Add the extended ownership voucher to the reseller database table `RT_DEVICES` using reseller API `POST /api/v1/resell/vouchers/<serial_number>`.
 
-6. Complete Transfer Ownership 1 and 2 (TO1 and TO2) by starting the FIDO IoT HTTP Java Device Sample again. The previously created 'credential.bin' from Step#2 will be used directly by the Device.
+6. If the reseller key to be used for voucher extension is not present in `RT_CUSTOMER` table, add it using reseller API `POST /api/v1/resell/keys/?alias=<keystore_alias>`. By default, the next Owner keys are present in the the table `RT_CUSTOMERS`, that are same as the keys present in Owner's keystore.
```

Comment:
Please remove redundant 'the', line 117.

Reply:
Fixed.